app/src/main/java/com/dankideacentral/dic/ConstantValues.java

```diff
@@ -0,0 +1,20 @@
+package com.dankideacentral.dic;
+
+/**
+ * Created by basim on 10/6/16.
+ * This file was part of the tutorial I was using. 
+ *
+ * TODO: Use SharedPreferences instead of this shitty file.
+ */
+
+public class ConstantValues {
```

Comment:
I was initially thinking of making second pull request to fix that issue, but perhaps it's better for me to commit to this feature until I'm officially done with the LoginActivity issue.